addons/mass_mailing/views/mailing_mailing_views.xml

```diff
@@ -187,28 +186,30 @@
                                     attrs="{'readonly': [('state', 'in', ('sending', 'done'))]}"/>
                                 </div>
                             </div>
-                            <label for="schedule_type" string="Schedule"/>
-                            <div name="schedule_block">
-                                <div class="row">
-                                    <div class="col-xs-12 col-md-3">
-                                        <field name="schedule_type"/>
-                                    </div>
-                                    <div class="col-xs-12 col-md-3" attrs="{'invisible': [('schedule_type', '=', 'now')]}">
-                                        <field name="schedule_date" attrs="{'required': [('schedule_type', '=', 'scheduled')]}" />
-                                    </div>
-                                </div>
-                            </div>
                         </group>
                         <notebook>
                             <page string="Mail Body" name="mail_body">
-                                <field name="body_html" class="oe_read_only" widget="html"
-                                    options="{'cssReadonly': 'mass_mailing.iframe_css_assets_readonly'}"/>
-                                <field name="body_arch" class="o_mail_body oe_edit_only" widget="mass_mailing_html"
-                                    options="{
-                                        'snippets': 'mass_mailing.email_designer_snippets',
-                                        'cssEdit': 'mass_mailing.iframe_css_assets_edit',
-                                        'inline-field': 'body_html'
-                                }" attrs="{'readonly': [('state', 'in', ('sending', 'done'))]}"/>
+                                <div style="margin-top: -16px">
```

Comment:
Inline styling should be avoided in form views, we try to stick on standard odoo code. Use a class and some CSS if necessary, but no inline styling.